goben/server.go

```diff
@@ -230,31 +230,40 @@ func handleConnection(conn net.Conn, c, connections int, isTLS bool) {
 		return
 	}
 
-	go serverReader(conn, opt, c, connections, isTLS)
+	var wg sync.WaitGroup
+
+	wg.Add(1)
+	go serverReader(conn, opt, c, connections, isTLS, &wg, app.totalFlow)
```

Comment:
app.totalFlow should be opt.totalFlow encoded in the message instead.

Reply:
Also I think we should merge totalDuration and totalFlow somehow. I wonder about a good way to do that.